docs/intro.md

```diff
@@ -0,0 +1,94 @@
+# Sashimi
+
+<a href="url"><img 
+src="https://github.com/portugueslab/sashimi/blob/master/sashimi/icons/main_icon.png"
```

Comment:
[This image](https://portugueslab.github.io/sashimi/intro.html) does not seem to load correctly. Try without the hyperlink (`<a></a>`) tag:

```html
<img src=https://github.com/portugueslab/sashimi/blob/master/sashimi/icons/main_icon.png align="left" height="190" width="270">
```